### PR TITLE
Update changelog release6.3

### PR DESCRIPTION
**Pull request by CongMa13**
Update changelog for rocm release 6.3

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,28 +2,32 @@
...
 
-* Fixed a bug in randomized tensor input data generation
-* Various documentation formatting updates and fixes
 * Split kernel instances to improve build times
 
+### Resolved issues
+
+* Fixed a bug in randomized tensor input data generation.
+* Fixed default strides calculation to be in column major order.
+* Fixed small memory leak by properly destroying HIP event objects in tests.
```
Comment (amd-jnovotny) [inline CHANGELOG.md]:
```suggestion
* Fixed a small memory leak by properly destroying HIP event objects in tests.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,28 +2,32 @@
...
+### Optimized
 
-* Fixed a bug in randomized tensor input data generation
-* Various documentation formatting updates and fixes
 * Split kernel instances to improve build times
 
+### Resolved issues
+
+* Fixed a bug in randomized tensor input data generation.
+* Fixed default strides calculation to be in column major order.
```
Comment (amd-jnovotny) [inline CHANGELOG.md]:
```suggestion
* Fixed the default strides calculation to be in column major order.
```

**Review by amd-jnovotny — APPROVED** [review]
Looks good. Just a couple of minor editorial suggestions.

File: CHANGELOG.md (hunk 1 above)
Comment (CongMa13) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md (hunk 2 above)
Comment (CongMa13) [reply CHANGELOG.md]: updated

**Comment by CongMa13** [comment]
Hi @amd-jnovotny , the change log has been updated according to your suggestion. Could you approve it again :)

**Review by amd-jnovotny — APPROVED** [review]
looks good now

**Pull request closed without merging**